src/components/ui/Fader.scss

```diff
@@ -0,0 +1,28 @@
+.fader-container {
+  .fader {
+    -webkit-appearance: none; /* Override default CSS styles */
+    appearance: none;
+    width: 50%;
+    height: 2px; /* Specified height */
+    background: #493f3f;
+    outline: none; /* Remove outline */
+    opacity: 0.7; /* Set transparency (for mouse-over effects on hover) */
+    -webkit-transition: 0.2s; /* 0.2 seconds transition on hover */
+    transition: opacity 0.2s;
```

Comment:
by using the parent reference `&` you can nest `::-webkit-slider-thumb` under `.fader` like so: 

```sass
  .fader {
  ...
  &::-webkit-slider-thumb {
      -webkit-appearance: none;
      appearance: none;
      width: 20px;
      height: 50px;
      background: #493f3f;
      border: 1px solid black;
      cursor: pointer;
    }
  }
```

Reply:
done 👍